planty/posts/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from django.contrib.auth.models import User
+from datetime import datetime
+from django.core.validators import MinValueValidator
+from django.urls import reverse
+from planty.utils import crop_max_square
+from PIL import Image
+
+
+class Post(models.Model):
+
+    OUTDOOR = 'O'
+    INDOOR = 'I'
+    FLOWERS = 'F'
+    NO_CATEGORY = 'ALL'
```

Comment:
NIT, I would use another class here so that refered variable names are more clear, e.g.:
```python
class Category:
    OUTDOOR = 'O'
    INDOOR = 'I'
    FLOWERS = 'F'
    NO_CATEGORY = 'ALL'
```
which are then referred to as:
```python
# outside of the class
Post.Category.OUTDOOR
# from within the class
self.Category.OUTDOOR
```